doc/releases/changelog-dev.md

```diff
@@ -4,11 +4,60 @@
 
 <h3>New features since last release</h3>
 
+* One can now pass a `qml.measurements.SampleMeasurement`-based measurements into a snapshot. [(#5582)](https://github.com/PennyLaneAI/pennylane/pull/5582)
+
+```python
```

Comment:
Would be good to have these lines indented with two spaces to match the corresponding bullet point.